lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -125,16 +125,13 @@ private static Integer toInteger(Long value) {
             throws IOException, IllegalArgumentException {
```

Comment:
Is there documentation for these functions anywhere? If not, do you think it would be useful to add a docstring before the function definition to help understand what this function is doing?

Reply:
There is a todo for javadoc in the interface. Do we need it for private methods?